en/FlyView/FlyView.md

```diff
@@ -1,79 +1,375 @@
+
+##### Note: How Resume Mission Rebuilding Works
+
+A mission cannot simply resume from the last mission item the vehicle executed because there may be multiple items at the last waypoint that affect the next stage of the mission (e.g. speed commands or camera control commands).
+Skipping any mission items at the last waypoint may prevent the rest of the mission running correctly.
+
+Therefore *QGroundControl* rebuilds the mission looking backwards from the last mission item flown, and automatically appends relevant commands to the front of the mission. 
+By doing this the state of the mission prior to the resume point is restored.
+
+The following mission commands are the ones scanned for:
+* [MAV_CMD_DO_CONTROL_VIDEO](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_CONTROL_VIDEO)
+* [MAV_CMD_DO_SET_ROI](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI)
+* [MAV_CMD_DO_DIGICAM_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONFIGURE)
+* [MAV_CMD_DO_DIGICAM_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONTROL)
+* [MAV_CMD_DO_MOUNT_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE)
+* [MAV_CMD_DO_MOUNT_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL)
+* [MAV_CMD_DO_SET_CAM_TRIGG_DIST](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_DIST)
+* [MAV_CMD_DO_FENCE_ENABLE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FENCE_ENABLE)
+* [MAV_CMD_IMAGE_START_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_IMAGE_START_CAPTURE)
+* [MAV_CMD_IMAGE_STOP_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_IMAGE_STOP_CAPTURE)
+* [MAV_CMD_VIDEO_START_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_VIDEO_START_CAPTURE)
+* [MAV_CMD_VIDEO_STOP_CAPTURE](https://mavlink.io/en/messages/common.html#MAV_CMD_VIDEO_STOP_CAPTURE)
+* [MAV_CMD_DO_CHANGE_SPEED](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_CHANGE_SPEED)
+* [MAV_CMD_NAV_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF)
+
+
+#### Remove Mission Prompt After Landing {#resume_mission_prompt}
+
+You will be prompted to remove the mission from the vehicle after the mission completes and the vehicle lands and disarms. 
+This is meant to prevent issues where stale missions are unknowingly left on a vehicle, potentially resulting in unexpected behavior.
+
+### Display Video {#video_switcher}
+
+*QGroundControl* supports RTP and RTSP video streaming over your vehicle's UDP connection.
+When video streaming is enabled the stream is displayed in the video switcher window at the bottom left of the map.
+Click the switcher to toggle *Video* and *Map* to foreground.
+
+![Video Stream Record](../../assets/fly/video_record.jpg)
+
+> **Note** The vidoe stream is configured/enabled in [Application Settings > General tab > Video](../SettingsView/General.md#video).
+
+
+It is possible to resize the video switcher.
```

Comment:
@Williangalvani As far as I can see you can resize and hide the switcher, but you can't move it - is that correct?

Reply:
Not in there, only if you detach it:
![Peek 2019-03-18 10-54](https://user-images.githubusercontent.com/4013804/54535324-12f7bf80-496d-11e9-8d80-cde156cff9eb.gif)

Reply:
Awesome thanks! I have updated the section appropriately.